### PR TITLE
Update of README.md

### PR DESCRIPTION
**Pull request by hdthaido**
Hi, I saw that this project was open-sourced and wanted to contribute. I saw that your README.md didn't have a guide-line on how to contribute to this project so I updated it with a step-step guide on how to contribute. I've also attached photo's as well which are stored in the "docs" folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,119 @@
-# commit.dev-proto
```
Comment (julie-huang) [inline README.md:1]: We should keep this title

File: README.md [hunk 2]
```diff
@@ -1,2 +1,119 @@
...
-Marketing website for Commit (commit.dev)
+# Introduction
```
Comment (julie-huang) [inline README.md]:
```suggestion
# commit.dev-proto**
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,119 @@
...
+
+## Push changes to Github
+
+Push your changes using the `git push` command:
+
+```
+git push origin <add-your-branch-name>
+```
+
+replacing `<add-your-branch-name>` with the name of the branch you created earlier.
```
Comment (julie-huang) [inline README.md]: We may not need all these explicit instructions on git usage. 

Could be something more concise like: 
```
## Getting started
`git clone https://github.com/this-is-you/commit.dev-proto.git`
`npm install`
`npm run dev` 
```
And then add a link to the [Official Github Guide](https://guides.github.com/activities/forking/) for more information

File: README.md [hunk 4]
```diff
@@ -1,2 +1,119 @@
-# commit.dev-proto
-Marketing website for Commit (commit.dev)
+# Introduction
+Marketing website for Commit (commit.dev).
+
+You can find our staging website [here](https://landing.commit-staging.dev/).
+# How to Contribute
```
Comment (julie-huang) [inline README.md]: Perhaps you can link this out to the wiki with some simple instructions that you have below (create changes on separate branch, and create a PR for reviewers to review, etc) 😄

**Review by julie-huang — CHANGES_REQUESTED** [review]
Thanks for contributing to our README 🚀 !  Please take a look at the suggestions in the review and let me know if there are any questions that I can help clarify

File: README.md (hunk 1 above)
Comment (hdthaido) [reply README.md:1]: Hi Julie, thanks for the feedback, and I have removed the "Introduction" as the title and put it back in as  #commit.dev-proto

File: README.md (hunk 3 above)
Comment (hdthaido) [reply README.md]: Thanks again, I took your advice and kept the instructions simple using your example.

File: README.md (hunk 4 above)
Comment (hdthaido) [reply README.md]: I will try and work on this afterwards.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,19 @@
...
+git clone https://github.com/this-is-you/commit.dev-proto.git
+```
+
+```
+npm install
+```
+
+```
+npm run dev
+```
```
Comment (superkhau) [inline README.md:17]: How about:

```
git clone https://github.com/commitdev/commit.dev.git
npm install
npm start
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,19 @@
...
+
+For more information here's a link to the [Official Github Guide](https://guides.github.com/activities/forking/) for more information.
```
Comment (superkhau) [inline README.md]: Maybe this should be under a contribution section in the wiki?

**Review by superkhau — CHANGES_REQUESTED** [review]

**Review by superkhau — COMMENTED** [review]
Also, please squash your commits.

File: README.md (hunk 1 above)
Comment (jasondippel) [reply README.md:1]: Repo has recently changed names. This should be updated to reflect (should be `commit.dev` now)

**Review by superkhau — CHANGES_REQUESTED** [review]
Please remove all the pictures from this PR and fix the merge conflicts -- any lines with lines like `<<<<<<< HEAD`

**Pull request closed without merging**